scala3doc/resources/dotty_res/styles/scalastyle.css

```diff
@@ -159,63 +153,88 @@ th {
   width: 116px;
 }
 
-.sideMenuPart {
-  padding-left: 1em;
+/* Navigation */
+#sideMenu2 {
```

Comment:
Is there any clash with `sideMenu` name, that we use `sideMenu2`?

Reply:
I assume you remove old `sideMenu` styles and elements

Reply:
yes